autoscheduler/frontend/src/components/App/App.tsx

```diff
@@ -4,7 +4,7 @@ import { ThemeProvider } from '@material-ui/styles';
 
 import theme from '../../theme';
 import * as styles from './App.css';
-import NavBar from '../NavBar';
+import NavBar from '../NavBar/NavBar';
```

Comment:
Interesting, it looks like the file should exist but linting says it doesn't. Weird.

Reply:
Oh wait, it's the caps. Linux is case-sensitive, so we'll need to rename the folder from `Navbar` to `NavBar`

Reply:
I thought I fixed this by renaming the folder before I pushed but apparently not. How do I make sure it gets renamed correctly? I'm thinking of deleting the folder, pushing that, adding it again with the correct casing, and pushing that but that seems pretty roundabout.

Reply:
If you delete a file and add another identical one in the same commit, git will recognize that you renamed the file. That should be enough to fix this